packages/ra-core/src/form/FormField.tsx

```diff
@@ -21,20 +22,11 @@ interface Props extends Omit<FieldProps<any, HTMLElement>, 'validate'> {
 }
 
 export const FormField: SFC<Props> = ({ input, validate, ...props }) => {
```

Comment:
input is injected by the Field, it should not be in the props, should it?

Reply:
Sorry, disregard my comment

Reply:
It's here to check wether we need to inject it (for backward compatibility). If present, we don't need to inject the props from the hook